### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by mpusch88**
Replaced unsecure link on lemmings with a Wikipedia article.

Commented out some missing links, added simple git contribution instructions for the time being.

Note - Contributing.md is also currently broken

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Each Lemmy server can set its own moderation policy; appointing site-wide admins
 - Lead singer from [Motörhead](https://invidio.us/watch?v=pWB5JZRGl0U).
 - The old school [video game](<https://en.wikipedia.org/wiki/Lemmings_(video_game)>).
 - The [Koopa from Super Mario](https://www.mariowiki.com/Lemmy_Koopa).
-- The [furry rodents](http://sunchild.fpwc.org/lemming-the-little-giant-of-the-north/).
+- The [furry rodents](https://en.wikipedia.org/wiki/Lemming).
```
Comment (Nutomic) [inline README.md:66]: There is nothing wrong with this, a readonly page doesnt need https. And wikipedia is boring.

File: README.md [hunk 2]
```diff
@@ -139,7 +139,7 @@ Lemmy is free, open-source software, meaning no advertising, monetizing, or vent
 - [Support on Liberapay](https://liberapay.com/Lemmy).
 - [Support on Patreon](https://www.patreon.com/dessalines).
 - [Support on OpenCollective](https://opencollective.com/lemmy).
-- [List of Sponsors](https://join-lemmy.org/sponsors).
+<!-- - [List of Sponsors](https://join-lemmy.org/sponsors). -->
```
Comment (Nutomic) [inline README.md:142]: Dont comment it out but change to https://join-lemmy.org/donate

File: README.md [hunk 3]
```diff
@@ -150,13 +150,15 @@ Lemmy is free, open-source software, meaning no advertising, monetizing, or vent
 
 ## Contributing
 
-- [Contributing instructions](https://join-lemmy.org/docs/en/contributing/contributing.html)
+<!-- - [Contributing instructions](https://join-lemmy.org/docs/en/contributing/contributing.html)
 - [Docker Development](https://join-lemmy.org/docs/en/contributing/docker_development.html)
-- [Local Development](https://join-lemmy.org/docs/en/contributing/local_development.html)
+- [Local Development](https://join-lemmy.org/docs/en/contributing/local_development.html) -->
+
+- In order to contribute to Lemmy, you will need to fork this repository. You can then make changes to the code and submit a pull request. Changes will be reviewed and merged into the main repository if accepted.
```
Comment (Nutomic) [inline README.md:157]: Fix the links instead of commenting them out. Text should go directly in the docs.

File: README.md (hunk 1 above)
Comment (mpusch88) [reply README.md:66]: fair enough - considering this is a git readme it's probably not for general users.

as a note, i feel that having links with security errors on public facing sites can definitely scare away new users however!

**Review by dessalines — APPROVED** [review]
Thanks!

**Pull request merged**